### PR TITLE
Update ExponentialCutoffPowerLaw1D

### PR DESCRIPTION
**Pull request by sjhannon**
Update description to include Schecter function alias.

### CODE REVIEW

File: astropy/modeling/powerlaws.py [hunk 1]
```diff
@@ -390,7 +390,7 @@ def _parameter_units_for_data_units(self, inputs_unit, outputs_unit):
 
 class ExponentialCutoffPowerLaw1D(Fittable1DModel):
     """
-    One dimensional power law model with an exponential cutoff.
+    One dimensional power law model with an exponential cutoff (also referred to as the Schecter function).
```
Comment (pllim) [inline astropy/modeling/powerlaws.py]: Nitpick: Your addition makes the line too long. Would be nicer to have it on the next line.

File: astropy/modeling/powerlaws.py [hunk 2]
```diff
@@ -390,7 +390,8 @@ def _parameter_units_for_data_units(self, inputs_unit, outputs_unit):
 
 class ExponentialCutoffPowerLaw1D(Fittable1DModel):
     """
-    One dimensional power law model with an exponential cutoff.
+    One dimensional power law model with an exponential cutoff, also 
+    referred to as the Schecter function.
```
Comment (larrybradley) [inline astropy/modeling/powerlaws.py]: Schecter -> Schechter

File: astropy/modeling/powerlaws.py [hunk 3]
```diff
@@ -390,7 +390,8 @@ def _parameter_units_for_data_units(self, inputs_unit, outputs_unit):
 
 class ExponentialCutoffPowerLaw1D(Fittable1DModel):
     """
-    One dimensional power law model with an exponential cutoff.
+    One dimensional power law model with an exponential cutoff, also
```
Comment (larrybradley) [inline astropy/modeling/powerlaws.py]: Remove the extra space at the end of the line (this is why `travis` is failing).

**Comment by larrybradley** [comment]
Thanks, @sjhannon, but please squash into a single commit for this small change.

**Pull request closed without merging**